pkg/devfile/sample-placeholder.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "name": "nodejs-basic",
+        "displayName": "Basic NodeJS",
+        "description": "A simple Hello world NodeJS application",
+        "type": "sample",
```

Comment:
What does this `type` rerpresent?

Reply:
The type is more for our main registry support. In our registry, we support stacks (generic project type support) and samples. ODC currently only supports samples so all the items return to you are sample type only. We reuse the same json for registry query and that's why that field is there. You can safely ignore that since when we do the registry query, we'll apply filter to only return samples in the ODC use case.